### PR TITLE
Style the claim form

### PR DESCRIPTION
**Pull request by whit537**
Part of #4305, follows on #4406.

- [x] style the confirmation form so it's not so plain
- [x] highlight/preselect any already-verified email(s) on the confirmation form
  - [x] list emails in an order: primary, verified, linked-but-unverified, unlinked, linked to another account
  - [x] style the list

### CODE REVIEW

**Comment by whit537** [comment]
Squashed and rebased on #4406, was 07d7e81ae65bed943f1b3e54ce65be4e5a4ba774.

**Comment by whit537** [comment]
p7qgv8l0gs

**Comment by whit537** [comment]
Rebased, was a3705b138d02207aa2fca33114c66858177572da.

File: gratipay/models/package/emails.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        for email in participant.get_emails():
+            if email.address == participant.email_address:
+                group = PRIMARY
+            elif email.verified:
+                group = VERIFIED
+            else:
+                group = UNVERIFIED
+            group_by_participant_email[email.address] = group
+
+        other_verified = self.db.all('''
```
Comment (dowski) [inline gratipay/models/package/emails.py]: This looks like another one of those reverse-mixins.

File: js/gratipay/dropdown.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
+Gratipay.Dropdown = function(selector) {
+    var $ul = $(selector);
+    var $labels = $('label', $ul);
+
+    // state for vertical position
+    var topFactor = 0;      // float between 0 and $labels.length-1
+    var maxTopFactor = $labels.length - 1;
```
Comment (dowski) [inline js/gratipay/dropdown.js]: What if $labels.length is 0?

File: js/gratipay/dropdown.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        if (scrollThrottle !== null) return;
+        window.setTimeout(function() {
+            var t = topFactor, by = e.deltaY / 128; // divisor arrived at experimentally
+            by = by * e.deltaFactor;
+            t = t - by;
+            if (t < 0) t = 0;
+            if (t > maxTopFactor) t = maxTopFactor;
+            moveTo(t);
+            scrollThrottle = null;
+        }, 12);
```
Comment (dowski) [inline js/gratipay/dropdown.js]: Why 12?

File: gratipay/models/package/emails.py (hunk 1 above)
Comment (whit537) [inline gratipay/models/package/emails.py]: I guess let's discuss the general pattern at https://github.com/gratipay/gratipay.com/pull/4422#discussion_r113821151.

File: js/gratipay/dropdown.js (hunk 2 above)
Comment (whit537) [inline js/gratipay/dropdown.js]: I believe that's prevented by the condition in the template at https://github.com/gratipay/gratipay.com/blob/371e17a8e089515cb4234e9b7c361de95df02fa1/www/on/npm/%25package/index.html.spt#L61, no? If there are no emails then there is no `.gratipay-dropdown` rendered and therefore we never get here.

File: js/gratipay/dropdown.js (hunk 3 above)
Comment (whit537) [inline js/gratipay/dropdown.js]: Commented in 47ad781406bbc5d1bec5e12f5fc8bcb08a950177. S'okay?

File: js/gratipay/dropdown.js (hunk 2 above)
Comment (dowski) [inline js/gratipay/dropdown.js]: Right!

**Pull request merged**